### PR TITLE
Remove Useless Toasts and Other Minor Changes

### PR DESCRIPTION
**Pull request by Tonnie-Dev**
- Added a Toast to `openWebPage()` function on **ContextExt.kt** file
- Removed the Toasts which should be on the caller's end
- Introduced idiomatic code on `selectAllThisAccount()` on `AccountSelectionViewModel`
- Correct preview rendering issue on CategoryTransactionListScreenContent 
- Added an intermediate compose screen to correct _Failed to Instantiate ViewModel Rendering Error_.
preview

### CODE REVIEW

File: core/designsystem/src/main/kotlin/com/naveenapps/expensemanager/core/designsystem/ui/extensions/ContextExt.kt [hunk 1]
```diff
@@ -29,6 +30,9 @@ fun openWebPage(context: Context, webpage: String) {
         context.startActivity(intent)
     } catch (e: ActivityNotFoundException) {
         // Define what your app should do if no activity can handle the intent.
+        Toast.makeText(context, context.getString(R.string.no_browser_found), Toast.LENGTH_SHORT)
```
Comment (nkuppan) [inline core/designsystem/src/main/kotlin/com/naveenapps/expensemanager/core/designsystem/ui/extensions/ContextExt.kt:33]: Hey @Tonnie-Dev, This method does not handle the browser. It seems and Am I missing anything here?

File: core/designsystem/src/main/kotlin/com/naveenapps/expensemanager/core/designsystem/ui/extensions/ContextExt.kt [hunk 2]
```diff
@@ -40,6 +44,9 @@ fun openEmailToOption(context: Context, emailId: String) {
         context.startActivity(intent)
     } catch (e: ActivityNotFoundException) {
         // Define what your app should do if no activity can handle the intent.
+        Toast.makeText(context, context.getString(R.string.no_email_app_found), Toast.LENGTH_SHORT)
```
Comment (nkuppan) [inline core/designsystem/src/main/kotlin/com/naveenapps/expensemanager/core/designsystem/ui/extensions/ContextExt.kt:47]: Hey @Tonnie-Dev, I appreciate your effort. Would returning the error message and handling it on the client/caller end make sense?

Comment (Tonnie-Dev) [reply core/designsystem/src/main/kotlin/com/naveenapps/expensemanager/core/designsystem/ui/extensions/ContextExt.kt:47]: @Tonnie-Dev Actually you are right, this is not the correct approach, I will edit my commits

Comment (Tonnie-Dev) [reply core/designsystem/src/main/kotlin/com/naveenapps/expensemanager/core/designsystem/ui/extensions/ContextExt.kt]: By the way have you set out [Buy Me Coffee](https://www.buymeacoffee.com/) Account, Have learnt great deal from this repo and I would like to thank you personally 🙏🏽

**Review by nkuppan — APPROVED** [review]

**Pull request merged**